content/kb/troubleshooting-rwx-volume-fails-to-attached-caused-by-protocol-not-supported.md

```diff
@@ -20,19 +20,25 @@ Oct 11 07:42:23 dev-worker-1 k3s[1294]: Mounting arguments: mount -t nfs -o vers
 Oct 11 07:42:23 dev-worker-1-autoscaled-small-19baf778f50efd8c k3s[1294]: Output: mount.nfs: Protocol not supported for 10.43.207.185:/pvc-13538170-4278-4467-b2b0-1f1ba6f54a4c on /var/lib/kubelet/plugins/kubernetes.io/csi/driver.longhorn.io/185c34f566c2eca6e8c7c6a2ede2094c076d7d25ddae286dc633eeef80551af0/globalmount
 ```
 
-The issue applies to RWX volumes on hosts running OpenSUSE MicroOS (from 2023/10/08 to 2023/10/21) and other distributions using version 6.5.6 of the Linux kernel.
+The issue applies to RWX volumes on hosts running operating systems that use specific Linux kernel versions with known NFS-related bugs. Among the affected are OpenSUSE MicroOS (from 2023/10/08 to 2023/10/21) and other distributions using kernel version 6.5.6.
```

Comment:
@c3y1huang we can have this kind of check in our CLI in the future. This can improve the user experience when they need to check if they encounter any known issues, as they can use CLI to check the issues at runtime immediately via an automation way.

Reply:
ref: https://github.com/longhorn/longhorn/issues/7927 

cc @longhorn/dev

Reply:
Each of the checks that can change dynamically should have node Conditions that can be queried at any time.  That's the ask in #7931.